pkg/controllers/tagging/tagging_controller.go

```diff
@@ -135,19 +143,34 @@ func (tc *Controller) process() bool {
 		workItem, ok := obj.(*workItem)
 		if !ok {
 			tc.workqueue.Forget(obj)
-			utilruntime.HandleError(fmt.Errorf("expected workItem in workqueue but got %#v", obj))
+			err := fmt.Errorf("expected workItem in workqueue but got %#v", obj)
+			utilruntime.HandleError(err)
+			recordWorkItemErrorMetrics("dequeue_work_item")
```

Comment:
I don't think we need to emit a latency metric here since this means that the workitem is invalid, right?